### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Add styling
- Ran linters check
- Update README

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -39,46 +72,75 @@ input::placeholder {
...
 }
 
 .scoreboard {
-  border: 3px solid black;
-  margin: 20px 0;
-  width: 70%;
+  border: 3px solid rgb(196, 37, 37);
+  margin: 20px;
+  width: 90%;
 }
```
Comment (ichala) [inline src/styles.css]: - [x] there's a small issue when i refresh the scores i can not see all of them or scroll, kindly fix this issue

image


**_i suggest updating this class :_**

```CSS
.scoreboard {
    border: 3px solid rgb(196, 37, 37);
    margin: 20px;
    width: 90%;
   /*Add max-height and set the overflow to auto */
    max-height: 220px;
    overflow: auto;
}
```

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- App works as it supposes to be✔️ 
- Good Readme ✔️ 


 ## Required Changes ♻️


- [x] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles.css (hunk 1 above)
Comment (joshualyngdoh) [reply src/styles.css]: Thank you for the feedback.

**Review by arslanbisharat — APPROVED** [review]
Hi @joshualyngdoh,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you.

**Pull request merged**